stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/header/ChannelListHeader.kt

```diff
@@ -170,8 +170,8 @@ internal fun DefaultChannelListHeaderAction(
     ) {
         Icon(
```

Comment:
Edit seems weird. We cant anything without a context (a selected channel). That's why renamed to "New chat" button.

Reply:
That makes sense, it's kind of a weird situation currently, because the button doesn't do much at the moment, as we don't support creating a new channel or editing or anything. 

We could always keep it as hidden by default but still expose the slot. 

What do you think?

Reply:
Yes, every button without default action should be hidden: the "new chat" button in the toolbar, the "view info" button in the channel info dialog, etc.